modules/hostap/src/supp_api.c

```diff
@@ -348,6 +348,10 @@ static inline int chan_to_freq(int chan)
 static inline enum wifi_frequency_bands wpas_band_to_zephyr(enum wpa_radio_work_band band)
 {
 	switch (band) {
+	case WPA_KEY_MGMT_IEEE8021X:
+	case WPA_KEY_MGMT_IEEE8021X_SUITE_B:
+	case WPA_KEY_MGMT_IEEE8021X_SUITE_B_192:
+		return WIFI_SECURITY_TYPE_EAP_TLS;
```

Comment:
hey, this currently breaking CI

```
ccache /opt/toolchains/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DCONFIG_CTRL_IFACE -DCONFIG_CTRL_IFACE_ZEPHYR -DCONFIG_ECC -DCONFIG_MBO -DCONFIG_NO_CONFIG_BLOBS -DCONFIG_NO_CONFIG_WRITE -DCONFIG_NO_RANDOM_POOL -DCONFIG_ROBUST_AV -DCONFIG_RRM -DCONFIG_SAE -DCONFIG_SHA256 -DCONFIG_SHA384 -DCONFIG_SHA512 -DCONFIG_SME -DCONFIG_SUITEB192 -DCONFIG_WMM_AC -DCONFIG_WNM -DKERNEL -DK_HEAP_MEM_POOL_SIZE=151024 -DMBEDTLS_CONFIG_FILE=\"config-tls-generic.h\" -DNRF5340_XXAA_APPLICATION -DNRF70_ANT_GAIN_2G=0 -DNRF70_ANT_GAIN_5G_BAND1=0 -DNRF70_ANT_GAIN_5G_BAND2=0 -DNRF70_ANT_GAIN_5G_BAND3=0 -DNRF70_BAND_2G_LOWER_EDGE_BACKOFF_DSSS=0 -DNRF70_BAND_2G_LOWER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_2G_LOWER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_2G_UPPER_EDGE_BACKOFF_DSSS=0 -DNRF70_BAND_2G_UPPER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_2G_UPPER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_1_LOWER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_1_LOWER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_1_UPPER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_1_UPPER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_2A_LOWER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_2A_LOWER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_2A_UPPER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_2A_UPPER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_2C_LOWER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_2C_LOWER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_2C_UPPER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_2C_UPPER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_3_LOWER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_3_LOWER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_3_UPPER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_3_UPPER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_4_LOWER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_4_LOWER_EDGE_BACKOFF_HT=0 -DNRF70_BAND_UNII_4_UPPER_EDGE_BACKOFF_HE=0 -DNRF70_BAND_UNII_4_UPPER_EDGE_BACKOFF_HT=0 -DNRF70_DATA_TX -DNRF70_LOG_VERBOSE -DNRF70_MAX_TX_PENDING_QLEN=18 -DNRF70_MAX_TX_TOKENS=10 -DNRF70_PCB_LOSS_2G=0 -DNRF70_PCB_LOSS_5G_BAND1=0 -DNRF70_PCB_LOSS_5G_BAND2=0 -DNRF70_PCB_LOSS_5G_BAND3=0 -DNRF70_REG_DOMAIN=00 -DNRF70_RPU_PS_IDLE_TIMEOUT_MS=10 -DNRF70_RX_MAX_DATA_SIZE=1600 -DNRF70_RX_NUM_BUFS=48 -DNRF70_STA_MODE -DNRF70_SYSTEM_MODE -DNRF70_TCP_IP_CHECKSUM_OFFLOAD -DNRF_SKIP_FICR_NS_COPY_TO_RAM -DNRF_WIFI_AP_DEAD_DETECT_TIMEOUT=20 -DNRF_WIFI_IFACE_MTU=1500 -DNRF_WIFI_LOW_POWER -DPICOLIBC_DOUBLE_PRINTF_SCANF -DTLS_DEFAULT_CIPHERS=\"\"DEFAULT:!EXP:!LOW\"\" -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR__=1 -I/__w/zephyr/zephyr/kernel/include -I/__w/zephyr/zephyr/arch/arm/include -I/__w/zephyr/zephyr/modules/hostap/src -I/__w/zephyr/modules/lib/hostap -I/__w/zephyr/modules/lib/hostap/src/utils -I/__w/zephyr/modules/lib/hostap/src/drivers -I/__w/zephyr/modules/lib/hostap/src -I/__w/zephyr/zephyr/include -I/__w/zephyr/zephyr/include/net -I/__w/zephyr/zephyr/samples/net/wifi -I/__w/zephyr/zephyr/twister-out/thingy53_nrf5340_cpuapp/samples/net/wifi/sample.net.wifi.nrf7002eb/zephyr/include/generated/zephyr -I/__w/zephyr/zephyr/twister-out/thingy53_nrf5340_cpuapp/samples/net/wifi/sample.net.wifi.nrf7002eb/zephyr/include/generated -I/__w/zephyr/zephyr/soc/nordic -I/__w/zephyr/zephyr/include/zephyr/posix -I/__w/zephyr/zephyr/lib/posix/options/getopt -I/__w/zephyr/zephyr/soc/nordic/nrf53/. -I/__w/zephyr/zephyr/soc/nordic/common/. -I/__w/zephyr/zephyr/subsys/usb/device -I/__w/zephyr/zephyr/subsys/net/l2 -I/__w/zephyr/zephyr/twister-out/thingy53_nrf5340_cpuapp/samples/net/wifi/sample.net.wifi.nrf7002eb/zephyr/misc/generated -I/__w/zephyr/zephyr/subsys/shell/modules/kernel_service/thread/.. -I/__w/zephyr/zephyr/drivers/usb/common/nrf_usbd_common/. -I/__w/zephyr/zephyr/drivers/wifi/nrfwifi/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/utils/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/os_if/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/bus_if/bus/qspi/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/bus_if/bal/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/fw_if/umac_if/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/fw_load/mips/fw/inc -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/hw_if/hal/inc -I/__w/zephyr/zephyr/drivers/wifi/nrfwifi/src/qspi/inc -I/__w/zephyr/zephyr/subsys/net/ip -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/hw_if/hal/inc/fw -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/fw_if/umac_if/inc/fw -I/__w/zephyr/modules/hal/nordic/drivers/nrf_wifi/fw_if/umac_if/inc/default -I/__w/zephyr/modules/hal/cmsis/CMSIS/Core/Include -I/__w/zephyr/zephyr/modules/cmsis/. -I/__w/zephyr/modules/hal/nordic/nrfx -I/__w/zephyr/modules/hal/nordic/nrfx/drivers/include -I/__w/zephyr/modules/hal/nordic/nrfx/mdk -I/__w/zephyr/zephyr/modules/hal_nordic/nrfx/. -I/__w/zephyr/modules/lib/hostap/wpa_supplicant -I/__w/zephyr/modules/lib/hostap/src/common -I/__w/zephyr/modules/lib/hostap/src/eap_common -I/__w/zephyr/modules/lib/hostap/src/eap_server -I/__w/zephyr/modules/lib/hostap/src/radius -I/__w/zephyr/modules/lib/hostap/src/crypto -I/__w/zephyr/modules/lib/hostap/src/ap -I/__w/zephyr/modules/lib/hostap/src/rsn_supp -I/__w/zephyr/modules/lib/hostap/port/mbedtls -I/__w/zephyr/modules/crypto/mbedtls/include -I/__w/zephyr/zephyr/modules/mbedtls/configs -I/__w/zephyr/zephyr/modules/mbedtls/include -isystem /__w/zephyr/zephyr/lib/libc/common/include -Os -DNDEBUG -fno-strict-aliasing -Werror -Os -imacros /__w/zephyr/zephyr/twister-out/thingy53_nrf5340_cpuapp/samples/net/wifi/sample.net.wifi.nrf7002eb/zephyr/include/generated/zephyr/autoconf.h -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m33 -mthumb -mabi=aapcs -mfp16-format=ieee -mtp=soft --sysroot=/opt/toolchains/zephyr-sdk-0.16.8/arm-zephyr-eabi/arm-zephyr-eabi -imacros /__w/zephyr/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wdouble-promotion -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -fno-asynchronous-unwind-tables -ftls-model=local-exec -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/__w/zephyr/zephyr/samples/net/wifi=CMAKE_SOURCE_DIR -fmacro-prefix-map=/__w/zephyr/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/__w/zephyr=WEST_TOPDIR -ffunction-sections -fdata-sections --specs=picolibc.specs -D_POSIX_THREADS -std=c99 -MD -MT zephyr/CMakeFiles/zephyr.dir/modules/hostap/src/supp_api.c.obj -MF zephyr/CMakeFiles/zephyr.dir/modules/hostap/src/supp_api.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/modules/hostap/src/supp_api.c.obj -c /__w/zephyr/zephyr/modules/hostap/src/supp_api.c
/__w/zephyr/zephyr/modules/hostap/src/supp_api.c: In function 'wpas_band_to_zephyr':
/__w/zephyr/zephyr/modules/hostap/src/supp_api.c:355:9: error: duplicate case value
  355 |         case BAND_2_4_GHZ:
      |         ^~~~
/__w/zephyr/zephyr/modules/hostap/src/supp_api.c:351:9: note: previously used here
  351 |         case WPA_KEY_MGMT_IEEE8021X:
      |         ^~~~
/__w/zephyr/zephyr/modules/hostap/src/supp_api.c:352:9: error: case label value exceeds maximum value for type [-Werror=switch-outside-range]
  352 |         case WPA_KEY_MGMT_IEEE8021X_SUITE_B:
      |         ^~~~
/__w/zephyr/zephyr/modules/hostap/src/supp_api.c:353:9: error: case label value exceeds maximum value for type [-Werror=switch-outside-range]
  353 |         case WPA_KEY_MGMT_IEEE8021X_SUITE_B_192:
      |         ^~~~
```

could you follow up on it?

Reply:
@fabiobaltieri I sent a hotfix https://github.com/zephyrproject-rtos/zephyr/pull/79129

Reply:
thanks!